rest_framework-stubs/validators.pyi

```diff
@@ -2,6 +2,7 @@ from collections.abc import Callable, Container, Iterable, MutableMapping
 from typing import Any, Protocol, TypeVar
 
 from django.db.models import Model, QuerySet
+from django.utils.functional import _StrOrPromise
```

Comment:
Use `from django_stubs_ext import StrOrPromise`, which is public API.

Reply:
I checked django-stubs for that

Reply:
Yeah, that's internal to `django-stubs`. Other projects should import from `django_stubs_ext`

Reply:
https://github.com/typeddjango/djangorestframework-stubs/pull/621